content/src/fieldConfig/config.json

```diff
@@ -46,14 +46,10 @@
     "accountSuccessSnackbarBody": "You can now save your progress, access more features, and edit your business profile.",
     "accountSuccessSnackbarTitle": "Welcome to your Business.NJ.gov account!",
     "accountSuccessSnackbarTitleExistingAccount": "Your myNewJersey Account is Linked",
-    "needsAccountSnackbarTitle": "Link with myNewJersey to Save Your Progress",
-    "needsAccountSnackbarTitleExistingAccount": "Link Your myNewJersey Account to Save Your Progress",
     "needsAccountModalBody": "To use this feature, you need to complete your registration with the State of New Jersey's secure login system, MyNewJersey. This will ensure your privacy, and allow you to come back any time.",
-    "needsAccountSnackbarBodyExistingAccount": "Now that you’ve filled out your business information, you can save your progress and come back later by [linking your myNewJersey account](/account-setup?source=guest_snackbar). ",
     "needsAccountModalSubText": "Already have an account? [Log In](#)",
     "needsAccountModalButtonText": "Link with myNewJersey",
-    "needsAccountModalTitle": "Link with myNewJersey to Save Your Progress",
-    "needsAccountSnackbarBody": "Now that you've filled out your business information, you can save your progress [link your Business.NJ.gov account with myNewJersey](/account-setup?source=guest_snackbar). "
```

Comment:
Don't you need this? I was looking at the designs and it seems like this specific thing is? Or am I misread the designs.
<img width="618" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/2971c2e0-82db-4aa5-abbd-85470fa051bb">

Reply:
This copy is the copy for the `NeedsAccountSnackBar` that we no longer need. The copy for the cards lives in their respective markdown files.